scripts/test-binary.sh

```diff
@@ -14,16 +14,20 @@ function get_help {
     echo "  t | tags 'tags list'   Sets the tags to use"
     echo "  c | clear              Clears list of run files"
     echo "  p | pause              Pause after each stage"
+    echo "  n | names 'names list' List of file names to test [QUOTES NEEDED]"
     echo "  d | dry                Dry-run: show commands without executing them"
     echo "  v | verbose            Gives more info"
     echo ""
-    echo "If no options are given, it runs all the tests in test-artifacts"
+    echo "If no options are given, it runs all the vcd*.tf tests in test-artifacts"
```

Comment:
The `vcd` prefix has been around for months.
The original reason it is there is because the prefix is part of the function name when creating the Terraform files from tests.
Changing it at this stage will require careful refactoring and I am not sure it's a good idea, as the files will lose the connection with the package.
I suggest leaving it as it is.